core/src/main/java/hudson/Launcher.java

```diff
@@ -295,7 +297,9 @@ public ProcStarter stdout(@CheckForNull OutputStream out) {
          * @return {@code this}
```

Comment:
It makes sense to explicitly set the requirement that the class is remotable in Javadoc

Reply:
All `TaskListener` implementations are expected to be remotable but I will clarify this in Javadoc.